pkg/ebpf/tracer.go

```diff
@@ -418,17 +418,17 @@ func (t *Tracer) getTCPStats(mp *bpflib.Map, tuple *ConnTuple, seen map[ConnTupl
 	pid := tuple.pid
 	tuple.pid = 0
 
-	// This is required to avoid (over)reporting stats for connections sharing the same socket.
+	_ = t.m.LookupElement(mp, unsafe.Pointer(tuple), unsafe.Pointer(stats))
```

Comment:
Why was this moved up here? non-noticeable performance gain to short circuit if it was already seen?

Reply:
basically you can't do an early return anymore in the context of PID collision because we still want to fetch the `TCPStats` and return the RTT metrics.